celsos_bookSearch/celsos_bookSearch/Model/SearchResultResponse.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  SearchResultResponse.swift
+//  celsos_bookSearch
+//
+//  Created by Celso Junio Simões de Oliveira Santos on 21/05/21.
+//
+
+import Foundation
```

Comment:
It is important to keep the same code style and file structure across the app. Make sure to break one line after the import section. 

```
import Foundation

public struct SearchResultResponse: Decodable{
```